Turn/include/Enemy.h

```diff
@@ -26,7 +26,8 @@ enum EnemyType
     etSlimeball,
     etRatKing,
     etZabra,
-    etNumEnemyTypes   
+    etNumEnemyTypes,   
+	etGremlin
```

Comment:
Indentation

Reply:
This one looks really weird, it is spaced out properly on my local version, I'll look further into it

Reply:
Resolved in next commit